app/src/main/java/com/github/kulebin/myfoursquareapp/presenter/VenueDetailPresentation.java

```diff
@@ -0,0 +1,11 @@
+package com.github.kulebin.myfoursquareapp.presenter;
+
+import com.github.kulebin.myfoursquareapp.view.VenueDisplayData;
+
+public interface VenueDetailPresentation extends BasePresentation {
+
+    void presentVenueToShowData(VenueDisplayData venueToShowData);
+
+    String getVenueId();
```

Comment:
I don't like the way is used to pass id to use case, 
Let rethink this logic when we start UI integration